### ISSUE TITLE
Migration

### ISSUE THREAD

**Issue by mythmon** [mozilla/django-product-details#68]
In [a Normandy CI job](https://circleci.com/gh/mozilla/normandy/7483) we noticed that this library is missing a migration. Running `./manage.py makemigrations` creates a new migration in the library's directory.

**Comment by pmac** [comment]
Interesting. I use this in several projects and haven't seen this, but none are on Django 2.x. So my guess is that there's some change in the ORM that's asking for this. I'll see what I can figure out. If it's possible to paste the migration it wants to create for you in here that would be very helpful.

**Comment by mythmon** [comment]
I'm guessing the help text is the part that changed. I don't really see why that needs to be in the migration, but oh well. Here's the migration it produced:

```bash
$ > cat .direnv/python-3.7.1/lib/python3.7/site-packages/product_details
```
```python
# Generated by Django 2.0.10 on 2019-01-28 21:40

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('product_details', '0002_auto_20151006_1348'),
    ]

    operations = [
        migrations.AlterField(
            model_name='productdetailsfile',
            name='last_modified',
            field=models.CharField(help_text='Value of Last-Modified HTTP header', max_length=50),
        ),
    ]
```

**Comment by pmac** [comment]
That's just silly. I've no idea why Django wants migrations for field properties that don't affect the DB. This hasn't even changed. The `models.py` file was last modified in 2015. The only difference I can see is that the initial migration has the help text as a bytes string:

https://github.com/mozilla/django-product-details/blob/7380594ad7650b1ba85c478963dec3fa7e18bc9a/product_details/migrations/0001_initial.py#L18

So strange.

**Comment by glogiotatidis** [comment]
I've seen this too. It's something that's Python 3 / Django 2 only so not sure if including this will break Python 2 installs.

**Comment by peterbe** [comment]
Can someone review: https://github.com/mozilla/django-product-details/pull/69 please. I don't know what permissions I have on this repo but I can't use the GitHub tools to ask for a review.

**Comment by pmac** [comment]
Just released `0.13.1`. Let me know if there are any issues with it. 

Cheers!

**Comment by peterbe** [comment]
It worked. I added django-product-details to a new/different project and makemigrations didn't trigger this time.
